comfy_cli/command/distribution.py

```
@@ -1146,3 +1153,113 @@ def delete_cmd(
+        renderer.error(code="distribution_definition_invalid", message=str(e), details={"path": from_})
+        raise typer.Exit(code=1) from e
+    client = _builder_client(renderer, builder_url)
+    dist = _builder_call(renderer, lambda: client.update_distribution(distribution_id, definition))
+    if renderer.is_pretty():
+        renderer.success(f"Updated distribution {distribution_id}")
+    renderer.emit(dist, command="distribution update", changed=True)
+
+
+@app.command("resolve", help="Resolve model filenames to public download candidates (HF/CivitAI).")
+@tracking.track_command("distribution")
+def resolve_cmd(
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)